src/stats/ecdf.jl

```diff
@@ -0,0 +1,69 @@
+function ecdf_values(ecdf::StatsBase.ECDF, npoints)
```

Comment:
Maybe this could be renamed to `ecdf_xvalues`? It was a bit confusing to me that this is a range of `x` values and does not encode the values of the cumulative distribution.